sdl_android/src/main/java/com/smartdevicelink/api/SdlManager.java

```diff
@@ -143,24 +147,30 @@ protected void initialize(){
 
 		this.fileManager = new FileManager(_internalInterface, context);
 		this.fileManager.start(subManagerListener);
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+			this.audioStreamManager = new AudioStreamManager(_internalInterface);
+			this.audioStreamManager.start(subManagerListener);
+		} else {
+			this.audioStreamManager = null;
+		}
 		/*
 		this.lockscreenManager = new LockscreenManager(lockScreenConfig, context, _internalInterface);
 		this.lockscreenManager.start(subManagerListener);
 		this.screenManager = new ScreenManager(_internalInterface, this.fileManager);
 		this.screenManager.start(subManagerListener);
 		this.videoStreamingManager = new VideoStreamingManager(context, _internalInterface);
 		this.videoStreamingManager.start(subManagerListener);
-		this.audioStreamManager = new AudioStreamManager(_internalInterface);
-		this.audioStreamManager.start(subManagerListener);
+
 		*/
 	}
 
 	private void dispose() {
 		this.permissionManager.dispose();
 		this.fileManager.dispose();
+		this.audioStreamManager.dispose();
```

Comment:
This needs an `if` block similar to Line 150

Reply:
Could we instead do https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0193-update-android-min-sdk.md ? My additions to l150 were to solve the issue of min-sdk 8 but I want to avoid a nullable manager though...